rust/ql/lib/codeql/rust/security/SensitiveData.qll

```diff
@@ -0,0 +1,86 @@
+/**
+ * Provides classes and predicates for identifying sensitive data.
+ *
+ * 'Sensitive' data is anything that should not be sent in unencrypted form. This library tries to
+ * guess where sensitive data may either be stored in a variable or produced by a method.
+ */
+
+import rust
+private import internal.SensitiveDataHeuristics
+private import codeql.rust.dataflow.DataFlow
+
+/**
+ * A data flow node that might contain sensitive data.
```

Comment:
```suggestion
 * A data-flow node that might contain sensitive data.
```

Reply:
We use "data flow" more commonly than "data-flow" at present I think.  I don't really mind, but I'd like to move towards greater consistency.

Reply:
You're absolutely right! I must've swapped it around in my mind over the holidays 😅